cmd/testdata/petstore_openapi.yaml

```diff
@@ -0,0 +1,77 @@
+---
+openapi: "3.0.3"
+info:
+  title: "Pet Store API"
+  version: "1.0.0"
+x-kuadrant:
+  route:
+    name: "petstore"
+    namespace: "petstore-ns"
+    hostnames:
+      - example.com
+    parentRefs:
+      - name: gw
+        namespace: gw-ns
+servers:
+  - url: https://example.io/v1
+paths:
+  /cat:
+    x-kuadrant:  ## Path level Kuadrant Extension
+      backendRefs:
+        - name: petstore
+          port: 80
+          namespace: petstore
+      rate_limit:
+        rates:
+          - limit: 1
+            duration: 10
+            unit: second
+        counters:
+          - request.headers.x-forwarded-for
+    get:  # Added to the route and rate limited
+      operationId: "getCat"
+      responses:
+        405:
+          description: "invalid input"
+    post:  # NOT added to the route
+      x-kuadrant:
+        disable: true
+      operationId: "postCat"
+      responses:
+        405:
+          description: "invalid input"
+  /dog:
+    get:  # Added to the route and rate limited
+      x-kuadrant:  ## Operation level Kuadrant Extension
+        backendRefs:
+          - name: petstore
+            port: 80
+            namespace: petstore
+        rate_limit:
+          rates:
+            - limit: 3
+              duration: 10
+              unit: second
+          counters:
+            - request.headers.x-forwarded-for
+      operationId: "getDog"
+      responses:
+        405:
+          description: "invalid input"
+    post:  # Added to the route, NOT rate limited, secured
+      x-kuadrant:  ## Operation level Kuadrant Extension
+        backendRefs:
+          - name: petstore
+            port: 80
+            namespace: petstore
+      operationId: "postDog"
+      security:
+        - securedDog: []
+      responses:
+        405:
+          description: "invalid input"
+components:
+  securitySchemes:
+    securedDog:
+      type: openIdConnect
+      openIdConnectUrl: https://example.com/.well-known/openid-configuration
```

Comment:
I know it's for the test data, but as an example the value used here can be dangerous. Please refer to https://github.com/Kuadrant/kuadrantctl/pull/76#discussion_r1592769112

Reply:
to be clear, you are suggesting `openIdConnectUrl` in this example to be something like `https://example.com` and authorino (using `go.oidc` lib)  will append automatically `/.well-known/openid-configuration` right? 

Then, we need to document betterl that `openIdConnectUrl` and what value are expected. Out of scope of this PR, btw.

Reply:
Looks like @azgabur will land a fix for this example in https://github.com/Kuadrant/kuadrantctl/pull/76/files

Reply:
Been addressed in https://github.com/Kuadrant/kuadrantctl/pull/76